config/schedule.rb

```diff
@@ -92,6 +92,14 @@
   rake_and_format "lookups:update:missing_address_attributes"
 end
 
+every :day, at: ENV["AREA_LOOKUP"] || "01:05", roles: [:db] do
+  rake_and_format "lookups:update:missing_ea_areas"
+end
+
+every :day, at: ENV["EASTING_NORTHING_LOOKUP"] || "03:30", roles: [:db] do
+  rake_and_format "lookups:update:missing_easting_northing"
+end
+
```

Comment:
@PaulDoyle-EA shouldn't these be the other way round? i.e. does the area lookup need an easting and northing?

Reply:
Logically yes, though the area lookup retrieves easting and northing in the same way as the missing_easting_northing job so it won't make any difference in practice. I'll switch them around for the sake of tidiness.